ui/app/components/variable-form.hbs

```diff
@@ -166,25 +154,9 @@
   <footer>
     {{#unless this.isJSONView}}
       {{#unless this.isJobTemplateVariable}}
-        <button
-          class="add-more button is-info is-inverted"
-          type="button"
-          disabled={{not (and this.keyValues.lastObject.key this.keyValues.lastObject.value)}}
-          {{on "click" this.appendRow}}
-          data-test-add-kv
-        >
-          Add More
-        </button>
+        <Hds::Button @text="Add More" @color="secondary" @size="medium" @icon="plus" class="add-more" type="button" disabled={{not (and this.keyValues.lastObject.key this.keyValues.lastObject.value)}} {{on "click" this.appendRow}} data-test-add-kv />
```

Comment:
This long `disabled` attr says "Don't let the user add more fields if their last one is currently empty"